main.js

```diff
@@ -1,73 +1,85 @@
-const bookListContainer = document.querySelector(".book-list");
-const form = document.querySelector(".form");
-
-const getBooks = () => JSON.parse(localStorage.getItem("books")) || [];
-
-const populateUI = () => {
-  let generalMarkup = ``;
-
-  //   Get latest books list
-  const books = getBooks();
-
-  books.forEach((book) => {
-    generalMarkup += `
-    <div class="books">
-                    <h3>${book.title}</h3>
-                    <p>${book.author}</p>
-                    <button class="remove-book-btn" data-id=${book.id}>Remove</button>
-                    <hr/>
-    </div>
-    `;
-  });
-
-  bookListContainer.insertAdjacentHTML("afterbegin", generalMarkup);
-};
-
-// POPULATE UI ON PAGE LOAD /RELOAD
-populateUI();
-
-// ADD BOOK
-const addBook = () => {
-  const formData = {};
-  form.addEventListener("submit", (e) => {
-    e.preventDefault();
-    formData.title = e.target.title.value;
-    formData.author = e.target.author.value;
-    formData.id = Date.now().toString();
-
-    // Get latest books arr and add new book
-    const books = getBooks();
-    books.push(formData);
-
-    // Update local storage - over ride the existing arr with new arr
-    localStorage.setItem("books", JSON.stringify(books));
-
-    // Refresh page
+class Books {
+  constructor() {
+    this.books = this.getBooks();
+    this.bookListContainer = document.querySelector(".book-list");
+    this.form = document.querySelector(".form");
+
+    this.populateUI();
+  }
+
+  getBooks() {
+    return JSON.parse(localStorage.getItem("books")) || [];
+  }
+
+  populateUI() {
+    let generalMarkup = ``;
+    this.books.forEach((book) => {
+      generalMarkup += `
+      <div class="books">
+          <p>"${book.title}" by ${book.author}</p>
+          <p><button class="remove-book-btn" data-id=${book.id}>Remove</button></p>
+      </div>
+      `;
+    });
+
+    if (this.books.length === 0) {
+      generalMarkup = `
+      <div class="no-books-container">
+      <h3>No books to display.</h3>
+      <p>Add a book.</p>
+      </div>
+      `;
+    }
+
+    this.bookListContainer.insertAdjacentHTML("afterbegin", generalMarkup);
+  }
+
+  addBook() {
+    const formData = {};
+    this.form.addEventListener("submit", (e) => {
+      e.preventDefault();
+      formData.title = e.target.title.value;
+      formData.author = e.target.author.value;
+      formData.id = Date.now().toString();
+
+      // Get latest books arr and add new book
+
+      this.books.push(formData);
+
+      // Update local storage - over ride the existing arr with new arr
+      localStorage.setItem("books", JSON.stringify(this.books));
+
+      // Refresh page
+      // eslint-disable-next-line no-restricted-globals
```

Comment:
- [ ] [OPTIONAL] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃